### PR TITLE
AP_Scripting: add documentation for DEBUG_LVL parameter

### PR DESCRIPTION
**Pull request by peterbarker**

### CODE REVIEW

File: libraries/AP_Scripting/AP_Scripting.cpp [hunk 1]
```diff
@@ -71,6 +71,11 @@ const AP_Param::GroupInfo AP_Scripting::var_info[] = {
     // @RebootRequired: True
     AP_GROUPINFO("HEAP_SIZE", 3, AP_Scripting, _script_heap_size, SCRIPTING_HEAP_SIZE),
 
+    // @Param: DEBUG_LVL
+    // @DisplayName: Enable Scripting Debug
+    // @Description: Enable scripting debug.
+    // @User: Advanced
+    // @RebootRequired: False
```
Comment (rmackay9) [inline libraries/AP_Scripting/AP_Scripting.cpp]: not that it matters but we don't normally add the RebootRequired if it's false.

Comment (rmackay9) [reply libraries/AP_Scripting/AP_Scripting.cpp]: We could add Values of 1:Default, 2:Enabled but this is a bit of a guess.  From a look at the code the only check is for ">1".

Comment (WickedShell) [reply libraries/AP_Scripting/AP_Scripting.cpp]: It's a threshold thing, it's supposed to get more verbose as it gets higher, but apparently I've removed all the stuff that used to only kick in at 1... I might actually bump the prompt of "No scripts to run" down to the level 1 area.

**Comment by peterbarker** [comment]
DevCallTopic in case it gets forgotten :-)

**Comment by rmackay9** [comment]
WickedShell approved it and it's certainly an improvement so merged!

**Pull request merged**